api/v1beta1/manila_webhook.go

```diff
@@ -61,6 +62,7 @@ func SetupDefaults() {
 		ShareContainerImageURL:     util.GetEnvVar("RELATED_IMAGE_MANILA_SHARE_IMAGE_URL_DEFAULT", ManilaShareContainerImage),
 		DBPurgeAge:                 DBPurgeDefaultAge,
 		DBPurgeSchedule:            DBPurgeDefaultSchedule,
+		APITimeout: APIDefaultTimeout,
```

Comment:
nit: Please align the `APIDefaultTimeout` with the rest of the values

Reply:
sorry, thank you! Done!